concordium-node/src/grpc2.rs

```diff
@@ -1086,7 +1086,22 @@ pub mod server {
                     }
                 });
                 let service = service::queries_server::QueriesServer::new(server);
-                let log_layer = tower_http::trace::TraceLayer::new_for_grpc();
+                let log_layer = {
+                    use tower_http::classify::GrpcCode as Code;
+                    // Change the trace level for gRPC status codes
+                    let classifier = tower_http::classify::GrpcErrorsAsFailures::new()
+                        .with_success(Code::Cancelled)
```

Comment:
It is not clear to me That Code::Ok is per default added to this list. Can you just add it?